resume-buddy/src/main/java/com/google/sps/servlets/BlobstoreUploadServlet.java

```diff
@@ -17,10 +17,21 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String redirectUrl = (String) request.getParameter("redirect");
+    if (redirectUrl == null) {
+      redirectUrl = "/reviewee-data";
+    }
     BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
-    String uploadUrl = blobstoreService.createUploadUrl("/reviewee-data");
+    String uploadUrl = blobstoreService.createUploadUrl(redirectUrl);
 
     response.setContentType("text/html");
     response.getWriter().println(uploadUrl);
   }
+
+  public static String getUploadUrl() {
+    BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
+    String uploadUrl = blobstoreService.createUploadUrl("/index.html");
+
+    return uploadUrl;
+  }
```

Comment:
Is this method needed? I don't see it called anywhere.

Reply:
Forgot to delete